### DISCUSSION TITLE
Removal of default command list

### DISCUSSION THREAD

**Discussion by helander** [pocketbase/pocketbase#3061] [Q&A] [answered]
When you run pocketbase without any command or just --help, the output shows a list of all available commands.
In this situation I would like to have the possibility to only show a list of my own custom commands. I have tried to remove the subcommands of $app.rootCommand using the removeCommand() method (in the before boostrap handler), but that did not work.

This is not a show stopper for me, but it would have been nice to be able to provide a better user experience for non-experienced users.

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand your use case but you can try to override the default usage template with something like `$app.rootCmd.setUsageTemplate("Example how to use...")` _(it is safe to define it at root/top level, without a hook)_.

**Reply by helander** [reply]
Do you by root level mean at top level of a *.pb.js file in the hook directory?

**Reply by ganigeorgiev** [reply, marked as answer]
Yes (_I've updated it to be more clear_).

**Reply by helander** [reply]
I got it to act, now I need to learn the template language :).

**Reply by helander** [reply]
Is there some place where the template language is described?

**Reply by ganigeorgiev** [reply]
The `$template` binding uses the standard Go template syntax.

For more information you can check:
- https://pkg.go.dev/text/template
- https://pkg.go.dev/html/template
- [Hashicorp's "Learn Go template syntax" tutorial](https://developer.hashicorp.com/nomad/tutorials/templates/go-template-syntax)
